src/corelibs/U2Core/src/util/U2SequenceUtils.cpp

```diff
@@ -67,7 +67,7 @@ qint64 U2SequenceUtils::length(const U2EntityRef &ref, U2OpStatus &os) {
 
 U2Sequence U2SequenceUtils::copySequence(const DNASequence &srcSeq, const U2DbiRef &dstDbi, const QString &dstFolder, U2OpStatus &os) {
     U2Sequence res;
-    res.alphabet = srcSeq.alphabet->getId();
+    res.alphabet = srcSeq.alphabet == nullptr ? nullptr : srcSeq.alphabet->getId();
```

Comment:
Does it possible? Maybe `SAFE_POINT` is better?

Reply:
Yes it is possible and looks like a big MSA related bug/feature.
Any new MSA object read from a ClustalW file has no per-sequence alphabet, but has only global per-MSA alphabet.

I decided not to change this behavior in my commit and address is separately.

Reply:
Added TODO, will fix this issue in the separate PR. Need to re-check all formats